autofs-5.0.9 - fix mistake in assignment

From: Ian Kent <ikent@redhat.com>

Fix an obvious mistake in an accumulation assignment.
---
 CHANGELOG    |    1 +
 lib/mounts.c |    4 ++--
 2 files changed, 3 insertions(+), 2 deletions(-)

--- autofs-5.0.5.orig/CHANGELOG
+++ autofs-5.0.5/CHANGELOG
@@ -194,6 +194,7 @@
 - fixes for samples/auto.master.
 - fix variable substitution description.
 - fix incorrect append options description in README.v5-release.
+- fix mistake in assignment.
 
 03/09/2009 autofs-5.0.5
 -----------------------
--- autofs-5.0.5.orig/lib/mounts.c
+++ autofs-5.0.5/lib/mounts.c
@@ -1929,7 +1929,7 @@ int clean_stale_multi_triggers(struct au
 		/* Check for and umount stale subtree offsets */
 		oe_base = oe->key + strlen(root);
 		ret = clean_stale_multi_triggers(ap, oe, root, oe_base);
-		left =+ ret;
+		left += ret;
 		if (ret)
 			continue;
 
@@ -1974,7 +1974,7 @@ int clean_stale_multi_triggers(struct au
 		key = strdup(oe->key);
 		if (!key) {
 	                char *estr = strerror_r(errno, buf, MAX_ERR_BUF);
-        	        error(ap->logopt, "malloc: %s", estr);
+		        error(ap->logopt, "malloc: %s", estr);
 			left++;
 			continue;
 		}
